galite-core/src/main/kotlin/org/kopi/galite/chart/VIntegerDimension.kt

```diff
@@ -15,7 +15,18 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
+
 package org.kopi.galite.chart
 
-class VIntegerDimension {
+/**
+ * Represents an integer chart column.
+ * @param ident The column identifier.
+ * @param format The dimension format.
+ */
+class VIntegerDimension(ident: String, format: VColumnFormat) : VDimension(ident, format) {
+  // --------------------------------------------------------------------
+  // IMPLEMENTATIONS
+  override fun toString(value: Any?): String {
+    return value?.toString() ?: CConstants.EMPTY_TEXT
```

Comment:
check if you can change value type to Any here and in mother class (VDimension) and make necessary changes